packages/react-button/src/components/ToggleButton/useChecked.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useControllableValue } from '@uifabric/react-hooks';
+
+export interface CheckedState {
+  checked?: boolean;
+  defaultChecked?: boolean;
+  onClick?: React.DOMAttributes<HTMLElement>['onClick'];
+
+  role?: string;
+  'aria-checked'?: boolean | 'false' | 'mixed' | 'true';
+  'aria-pressed'?: boolean | 'false' | 'mixed' | 'true';
```

Comment:
I suggest to reuse `React.AriaAttributes`

```suggestion
  'aria-pressed'?: React.AriaAttributes['aria-pressed'];
```

Reply:
good call!